lib/posthog/feature_flags.ex

```diff
@@ -0,0 +1,239 @@
+defmodule PostHog.FeatureFlags do
+  @type supervisor_name() :: PostHog.supervisor_name()
+  @type distinct_id() :: PostHog.distinct_id()
```

Comment:
this shorthand unfortunately will prevent hexdocs from rendering typedoc in a tooltip:

<img width="520" height="192" alt="image" src="https://github.com/user-attachments/assets/66de0432-263c-412c-a650-3bec0cb782cb" />

vs
<img width="689" height="303" alt="image" src="https://github.com/user-attachments/assets/7618a267-7aec-42d3-9c5c-8b33a18c7401" />

Reply:
Ah, I didn't know that, TIL! Thank you for using the explicit name